src/main/java/com/epam/tc/hw3/EpamIndexPage.java

```diff
@@ -0,0 +1,112 @@
+package com.epam.tc.hw3;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class EpamIndexPage {
+    @FindBy(className = "navbar-right")
+    private WebElement loginBar;
+
+    @FindBy(id = "name")
+    private WebElement name;
+
+    @FindBy(id = "password")
+    private WebElement pass;
+
+    @FindBy(id = "login-button")
+    private WebElement loginButton;
+
+    @FindBy(xpath = "//*[text()='Home']")
+    private WebElement headerHome;
+
+    @FindBy(xpath = "//*[text()='Contact form']")
+    private WebElement headerContact;
+
+    @FindBy(xpath = "//*[text()='Service']")
+    private WebElement headerService;
+
+    @FindBy(xpath = "//*[text()='Metals & Colors']")
+    private WebElement headerMetals;
+
+    @FindBy(className = "benefit-icon")
+    private List<WebElement> benefits;
+
+    @FindBy(className = "benefit-txt")
+    private List<WebElement> benefitsText;
+
+    @FindBy(id = "frame")
+    private WebElement frame;
+
+    @FindBy(id = "frame-button")
+    private WebElement frameButton;
+
+    @FindBy(className = "sidebar-menu")
+    private WebElement sideBar;
+
+    @FindBy(className = "navbar-nav")
+    private WebElement headerMenu;
+
+    @FindBy(xpath = "//*[text()='Different elements']")
+    private WebElement differentElements;
+
+    public EpamIndexPage(
+        WebDriver driver) {
+        PageFactory.initElements(driver, this);
+    }
+
+    public void clickToLoginBar() {
+        loginBar.click();
+    }
+
+    public void login(String userName, String userPassword) {
```

Comment:
Usually method `login` consist of 3 steps

1. Enter username

2. Enter password

3. Click login button

I coud see only 2 steps in your method